lib/stores.js

```diff
@@ -29,7 +29,14 @@ function stores(
   return (Component) => preparable(async function prepare(props, context) {
     const flux = context[fluxKey];
     const bindings = getBindings(props, flux);
-    const deps = _.map(bindings, (binding) => flux.fetchStore(binding));
+    const deps = _.map(bindings, (binding) => {
+      const state = flux.readStoreFromState(binding);
+      if(state.isResolved() || (state.isRejected() && !state.meta.uninitialized)) {
```

Comment:
I changed the State class so that meta defaults to empty object: b05b8fe97ac936e57b5658309f054835605c9b28

OK for you?